### DISCUSSION TITLE
Rainbowkit - WebView

### DISCUSSION THREAD

**Discussion by p2e-ggx** [rainbow-me/rainbowkit#720] [Q&A]
Does rainbowkit support integration inside a Web-View (Android App) ?

**Comment by SergioArrighi** [comment]
I'm trying this too.
Actually I'm using capacitor to build an Android app.
My app works fine in mobile web browser and it interacts with MetaMask mobile app just fine.
But when I run the build on my Android device (which is a bit old and doesn't have latest Android) the app crashes as soon as I tap the MetaMask icon.
Not sure if used with React Native would work.
Could someone confirm?

P.S.= sorry for necro

Thanks and regards

**Reply by SergioArrighi** [reply]
So I finally managed with capacitor (they had a bug in their code) so I guess the answer is yes.
Regards
